### PR TITLE
XS✔ ◾ Ai in UI design

### PR DESCRIPTION
**Pull request by lukecookssw**
**Tip: Use [SSW Rule Writer GPT](https://chat.openai.com/g/g-cOvrRzEnU-ssw-rules-writer) for help with writing rules 🤖**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

New rule needed for v0.dev (and alternatives) for CTF

> 2. What was changed?

New rule created. Added rule to GPT category

> 3. Did you do pair or mob programming (list names)?
I worked with @2fernandez, @GordonBeeming and @william-liebenberg

### CODE REVIEW

**Review by JackDevAU — APPROVED** [review]

**Review by GordonBeeming — APPROVED** [review]
🚢

**Review by william-liebenberg — APPROVED** [review]
💪

File: rules/do-you-use-ai-for-ui/rule.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+type: rule
+archivedreason: 
+title: Do You Use AI in UI Design?
```
Comment (tiagov8) [inline rules/do-you-use-ai-for-ui/rule.md:4]: Hye @lukecookssw 

- [ ] Please remove Capitalization from title (for consistency)

File: rules/do-you-use-ai-for-ui/rule.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+redirects: []
+
+---
+
+
+When a web developer is handed design images, it can be a drag to spend the next few hours (or days/weeks) building the baseline components, HTML, and CSS before getting to the fun stuff.
+
+Thankfully, AI-assisted tools are beginning to help with the more mundane aspects of this process, and sites such as [v0.dev](https://v0.dev) can produce React components you can drop into your project to give you a huge headstart on the work!
+
+`youtube: https://www.youtube.com/watch?v=By9wCB9IZp0`
```
Comment (tiagov8) [inline rules/do-you-use-ai-for-ui/rule.md:27]: Hey @lukecookssw 

- [ ] Please add a caption as per https://www.ssw.com.au/rules/add-useful-and-concise-figure-captions/#tip-5-add-the-titles--length-on-video-captions

File: rules/do-you-use-ai-for-ui/rule.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+If you're a dev that likes (or is forced) to moonlight as a designer these tools can be a great timesaver in creating professional looking web components in a short amount of time. These tools aren't a substitute for actual designers, but if your choice as a developer is between these tools or MS Paint, then you'll get a lot of value from them.
+
+## PoCs and Inspiration
+
+Generative UI tools can be a great way to collaborate with stakeholders in realtime, as you can generate ideas and iterate over them using natural language prompts. This can give developers or designers a useful starting point and frame of reference to build a highly polished design.
+
+### Tools
+
+[v0.dev](https://v0.dev)
```
Comment (tiagov8) [inline rules/do-you-use-ai-for-ui/rule.md:43]: Hey @lukecookssw 

- [ ] I would make this a list. Please action if agreed ;)

**Review by tiagov8 — CHANGES_REQUESTED** [review]

File: rules/do-you-use-ai-for-ui/rule.md (hunk 1 above)
Comment (lukecookssw) [reply rules/do-you-use-ai-for-ui/rule.md]: Done

File: rules/do-you-use-ai-for-ui/rule.md (hunk 2 above)
Comment (lukecookssw) [reply rules/do-you-use-ai-for-ui/rule.md:27]: Done

File: rules/do-you-use-ai-for-ui/rule.md (hunk 3 above)
Comment (lukecookssw) [reply rules/do-you-use-ai-for-ui/rule.md]: Done

**Comment by lukecookssw** [comment]
@tiagov8 thanks - I want you reviewing all my rule PRs from now on :)

**Comment by JackDevAU** [comment]
In Tiago we trust 🙏🙏

**Review by tiagov8 — APPROVED** [review]

**Pull request merged**